auth/generate-google-id-jwt2.py

```diff
@@ -0,0 +1,54 @@
+import json
+import logging
+import requests
+import time
+
+from pprint import pprint
+
+from google.auth import crypt, jwt
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+# generates a signed JSON Web Token
+
+with open('/usr/local/google/home/ensonic/robco_gob/cloud-robotics/src/bootstrap/robot/robot-credentials.json', 'r') as fh:
```

Comment:
do you want to hardcode the file path?

Reply:
Please use os.environ['GOOGLE_APPLICATION_CREDENTIALS'] or a dummy path like '/path/to/service_account.json'